scanpy/experimental/pl/_preprocessing.py

```diff
@@ -0,0 +1,111 @@
+from typing import Optional
+from anndata import AnnData
+from scanpy.plotting import scatter
+import numpy as np
+
+
+def pearson_residuals_hvg_scatter(
+    adata: AnnData,
+    marker_names=None,
+    gene_name_key: Optional[str] = None,
+    x: str = 'means',
+    y: str = 'residual_variances',
+    hvg_key: str = 'highly_variable',
+    kwargs_sc_pl_scatter: dict = dict(),
```

Comment:
can this just be called `kwargs` and in the docstrings specify that this is passed to `sc.pl.scatter`. ?